programs/clearing_house/src/controller/lp.rs

```diff
@@ -107,12 +107,10 @@ pub fn burn_lp_shares(
     //     .checked_sub(1)
     //     .ok_or_else(math_error!())?;
     // compute standardized + dust position in baa/qaa
-    let dust_base_asset_value = calculate_base_asset_value_with_oracle_price(
-        base_asset_amount,
-        oracle_price_data.price,
-    )?
-    .checked_add(1)
-    .ok_or_else(math_error!())?;
+    let dust_base_asset_value =
+        calculate_base_asset_value_with_oracle_price(base_asset_amount, oracle_price_data.price)?
```

Comment:
probs should just pass in the price directly since thats all we use from oracle_price_data